### PR TITLE
Examples were slightly outdated

### PR DESCRIPTION
**Pull request by ugocupcic**

### CODE REVIEW

File: sr_example/scripts/sr_example/link_joints.py [hunk 1]
```diff
@@ -32,23 +32,21 @@
 import rospy
 from std_msgs.msg import Float64
 # If you use the simulated hand (in gazebo) use the mixed controllers
-from sr_robot_msgs.msg import JointControllerState
+from pr2_controllers_msgs.msg import JointControllerState
 # If you use the real hand, generally use the position controller (comment the previous line and uncomment the following)
```
Comment (toliver) [inline sr_example/scripts/sr_example/link_joints.py]: Probably remove comments about real_hand simulated hand...?

File: sr_example/scripts/sr_example/shadowhand_subscriber.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ def listener():
     """
     rospy.init_node('shadowhand_joint_states_subscriber_python', anonymous=True)
     #For the simulated hand (running on the gazebo simulator)
-    rospy.Subscriber("gazebo/joint_states", JointState, callback)
+    rospy.Subscriber("/joint_states", JointState, callback)
     #For the real shadow hand
     #rospy.Subscriber("joint_states", JointState, callback)
```
Comment (toliver) [inline sr_example/scripts/sr_example/shadowhand_subscriber.py]: remove these comments about real/simulated

File: sr_example/scripts/sr_example/shadowhand_subscriber.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def listener():
     """
     rospy.init_node('shadowhand_joint_states_subscriber_python', anonymous=True)
     #For the simulated hand (running on the gazebo simulator)
-    rospy.Subscriber("gazebo/joint_states", JointState, callback)
+    rospy.Subscriber("/joint_states", JointState, callback)
```
Comment (toliver) [inline sr_example/scripts/sr_example/shadowhand_subscriber.py]: Remove leading slash /

Comment (ugocupcic) [inline sr_example/scripts/sr_example/shadowhand_subscriber.py]: fixed

On 14 February 2014 16:51, toliver notifications@github.com wrote:

> In sr_example/scripts/sr_example/shadowhand_subscriber.py:
> 
> > @@ -49,7 +49,7 @@ def listener():
> >      """
> >      rospy.init_node('shadowhand_joint_states_subscriber_python', anonymous=True)
> >      #For the simulated hand (running on the gazebo simulator)
> > -    rospy.Subscriber("gazebo/joint_states", JointState, callback)
> > -    rospy.Subscriber("/joint_states", JointState, callback)
> 
> Remove leading slash /
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/shadow-robot/sr-ros-interface/pull/88/files#r9752350
> .

## 

Ugo Cupcic _Senior Software Engineer_+44 20 7700 2487ugo@shadowrobot.com
www.shadowrobot.com

## 

---

_Shadow Robot Company Ltd._
251 Liverpool Road, N1 1LX, UK
Registered Number 3308007 (England & Wales)

RoNeX - Building Robots with ROS Made Easy 

## http://www.shadowrobot.com/products/ronex/

**Pull request merged**